smart_open/s3.py

```diff
@@ -170,44 +172,58 @@ def __init__(self, s3_object, content_length, version_id=None):
         self._object = s3_object
         self._content_length = content_length
         self._version_id = version_id
+        self._body_reload = True
```

Comment:
Do you really need this variable?

Isn't is possible to set `self._body = None`, and then expect `self._body is None` to mean the same thing as `self._body_reload is True`?

Reply:
you're right.